cmd/antrea-agent/agent.go

```diff
@@ -97,14 +97,19 @@ func run(o *Options) error {
 	}
 	defer ovsdbConnection.Close()
 
+	// Enable AntreaIPAM will set enableBridge to True. Currently only Linux+IPv4 is supported.
+	// AntreaIPAM works with system OVSDatapathType, noEncap and no Egress SNAT.
+	enableBridge := features.DefaultFeatureGate.Enabled(features.AntreaIPAM)
```

Comment:
What does "enableBridge" mean? If it is about moving uplink to bridge, probably name it "connectUplinkToBridge".

Reply:
Sound good. Will rename this.